pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -186,6 +186,13 @@ async def run(self):
             jira_issues = self.create_release_jira(jira_template_vars)
             jira_issue = jira_issues[0] if jira_issues else None
             jira_issue_key = jira_issue.key if jira_issue else None
+            subtask = self._jira_client.get_issue(jira_issue.fields.subtasks[1].key)
+            self._jira_client.add_comment(
+                    subtask,
+                    "prepare release job : {}".format(os.environ.get("BUILD_URL")),
+                    visibility={'type': 'group', 'value': 'Red Hat Employee'}
+            )
+            self._jira_client.transition_issue(subtask, '21')
```

Comment:
Would be good to un-magic string `21`.

Maybe store it in a variable with a good name, a link to where other numbers can be found?

Reply:
updated